### PR TITLE
Jordan+Fabo/1626 update sentry

### LINKED ISSUE #1626: Update Sentry library
Sentrys library changed:
https://docs.sentry.io/error-reporting/quickstart?platform=browser

### PR DESCRIPTION
**Pull request by jbibla**
Closes #1626 

_Description:_

- removed raven JS
- added sentry/browser

❤️ Thank you!

---

- [ ] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

**Comment by jbibla** [comment]
so it seems that the new sentry SDK has improved support for source maps - having a hard time understanding this documentation tbh. 

screen shot 2018-12-10 at 2 39 39 pm

**Comment by faboweb** [comment]
That looks awesome!

File: app/src/main/index.js [hunk 1]
```diff
@@ -399,11 +399,13 @@ const eventHandlers = {
   },
 
   "error-collection": (event, optin) => {
-    Raven.uninstall()
-      .config(optin ? config.sentry_dsn : ``, {
-        captureUnhandledRejections: false
+    if (optin) {
+      Sentry.init({
+        dsn: config.sentry_dsn,
+        release: `voyager@0.8001.1`
```
Comment (sabau) [inline app/src/main/index.js]: Yeah it's WIP but I was curious :) 
If you want to forget aligning this with package.json we can do something like

```javascript
import {version} from './package.json';
```

Comment (jbibla) [reply app/src/main/index.js]: good idea. will fix.

**Pull request merged**